source/extensions/filters/http/ext_proc/processor_state.cc

```diff
@@ -370,6 +370,10 @@ void ProcessorState::clearRouteCache(const CommonResponse& common_response) {
   if (filter_.config().disableClearRouteCache()) {
     filter_.stats().clear_route_cache_disabled_.inc();
     ENVOY_LOG(debug, "NOT clearing route cache, it is disabled in the config");
+  } else if (trafficDirection() == envoy::config::core::v3::TrafficDirection::OUTBOUND ) {
+    // Do not clear the route cache for outbound response traffic.
```

Comment:
```suggestion
    // Do not clear the route cache on the response path to downstream client.
```

nit: I think  `response path to downstream client`  is a bit clearer than outbound traffic as it is more widely used and more specific.  WDYT?

Reply:
Sounds good.